Fix visual bug in Molybdomancy

minhnhat1901 opened the pull request:
So sorry guys, I was working on fixing things in Molybdomancy and I realized at the end that I did in the wrong branch =((.

hpdinh committed:
#62 Yin-Yang Coin visual bug fix

Fixed container extending covering button

fillna committed:
Update JSON

hpdinh committed:
Changed instruction text to 8vh

10vh was too big and would still push the container down, so changed to 8vh

hpdinh committed:
Merge branch 'fix-visual-bug-yin-yang-coin' of https://github.com/cse110-sp23-group21/map-my-future into fix-visual-bug-yin-yang-coin

minhnhat1901 committed:
Molybdomancy #66

- Distinct the text of Shape and Meaning into 2 (HTML file) and add a color for each one (CSS file) -> More accessibility.
- Increase the font size of info, as well as make a info window a bit bigger, and increase an opacity a bit. (CSS file)
- Small change of the meaning in JSON file.
- Because I distinct the text, I need to add another variable in JS.

hpdinh committed:
Changed UI for better readibility

Changed opacity of containers and changed gray theme to purple for contrast

gilkeidar reviewed (changes requested):
I'm not sure about some of the color choices - the Molybdomancy text colors are a bit hard on the eyes, and I don't think the red / blue combination really works. The ideal thing is something that is easy / inviting to read. Maybe we can even add a slight background color to the text with a low opacity like in the Yin Yang Coin?

I'm also not sure why the Yin Yang Coin's colors were changed - I was thinking more of just raising the opacity of the background color. If we need to change the colors though to increase contrast, maybe something more earthy? The purple doesn't really work in my opinion.

gilkeidar committed:
Alternative colors for molybdomancy text

Changed text shadows to black

hpdinh committed:
Record and Header UI

Added counter to record and changed text shadow of header to be a lighter purple

gilkeidar committed:
Merge branch 'alternative-fix-molybdomancy' into fix-visual-bug-yin-yang-coin

gilkeidar reviewed (approved):
I think this is good to go

gilkeidar committed:
Merge branch 'dev' into fix-visual-bug-yin-yang-coin

gilkeidar committed:
Updated "melting tin" delay to fit new fade in time
